torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -472,14 +472,18 @@ def return_future():
 
 
 class RpcTest(RpcAgentTestFixture):
-    def _skip_if_tensorpipe_agent(old_func):  # noqa: B902
-        def decorator(self):
-            return unittest.skipIf(
-                self.rpc_backend == rpc.backend_registry.BackendType.TENSORPIPE,
-                "This test is not yet supported in the Tensorpipe Agent"
-            )(old_func)
+    def _skip_if_tensorpipe_agent(test_func):  # noqa: B902
+        @wraps(test_func)
+        def wrapped_test_func(self):
+            if self.rpc_backend == rpc.backend_registry.BackendType.TENSORPIPE:
+                # In theory we should call self.skipTest(reason), but this does
+                # not play well with the test runner, which sees an exception
+                # being raised and marks the test as failed. Thus instead we
+                # make the test a successful no-op.
+                return
+            test_func(self)
```

Comment:
should this be `return test_func(self)` in case the function returns anything?

Reply:
I was importing the accept into the internal diff and this triggered a land which didn't pick up this fix (as it was still a draft). If that's a problem I can send out a PR that fixes it. LMK.